.eslintrc.json

```diff
@@ -40,7 +40,8 @@
         "@typescript-eslint/ban-types": ["error"],
         "no-useless-escape": ["error"],
         "no-prototype-builtins": ["error"],
-        "prefer-spread": ["error"]
+        "prefer-spread": ["error"],
+        "@typescript-eslint/no-explicit-any": "off"
```

Comment:
It seems like this is enabled by default now, but there were too many issues to address.
Which would make this PR unclear and big, #1848 will resolve this and we can enable this rule back again within that PR.